### DISCUSSION TITLE
Quasar create new project with specific version

### DISCUSSION THREAD

**Discussion by samodraland** [quasarframework/quasar#9956] [General - Components / Directives / etc] [answered]
Hi All,

How to create quasar project is `quasar create projectname` right. I created new project, do copy-paste boot & some files from old project to the new created project. When it runs get error. After I check in package.json, quasar version in dependencies is v2 and @quasar/app in devDependencies is v3. 
fyi, I'm running Vue 2

How can I create project with specific quasar version?

**Comment by MilosPaunovic** [comment, marked as answer]
Hello @samodraland,

Check [this](https://v1.quasar.dev/quasar-cli/installation) out; but the command you're looking for is for creation of Quasar v1 project

```js
quasar create <folder_name> --branch v1
```

**Reply by samodraland** [reply]
Hi @MilosPaunovic Thank you. Yeah it works!
